templates/kapeta/block-type-service/ai-instructions.txt

```diff
@@ -0,0 +1,10 @@
+{{#ai-context}}
+{{ai-type 'instructions'}}
+//#FILENAME:STORM.md:write-never
+- You are coding java using the Spring Boot framework
+- Always use constructor injection instead of field injection.
```

Comment:
This is also in the ai-service, should it live both places?

Reply:
Nope - it'll go out of the AI service. Just dont want to have a PR in the AI-service longer than I have to :)